.github/workflows/docker-build.yaml

```diff
@@ -13,23 +13,17 @@ jobs:
       - name: "Checking out the repository"
         uses: actions/checkout@v2
       - name: "Calculating the docker image tag"
-        if: github.repository == 'geobretagne/mviewer' && github.ref == 'refs/heads/master'
-        id: version
-        run: |
-         echo ::set-output name=VERSION::stable
-
-      - name: "Calculating the docker image tag latest"
-        if: github.repository == 'geobretagne/mviewer' && github.ref == 'refs/heads/develop'
-        id: version
-        run: |
-         echo ::set-output name=VERSION::latest
-
-      - name: "Calculating the docker image tag latest"
-        if: github.repository == 'geobretagne/mviewer' && github.release.types == 'published'
+        if: github.repository == 'geobretagne/mviewer'
         id: version
         run: |
-         DOCKER_TAG=$( echo ${{ github.ref }} | cut -d'/' -f3)
-         echo ::set-output name=VERSION::$DOCKER_TAG
+          if [[ $GITHUB_REF == 'refs/heads/master' ]]; then
+            echo ::set-output name=VERSION::stable
+          elif [[ $GITHUB_REF == 'refs/heads/develop' ]]; then
+            echo ::set-output name=VERSION::latest
+          elif [[ ${{ github.release.types }} == "published" ]]; then
```

Comment:
Relying on [the doc](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#release), shouldn't it be "github.release.action" instead ? 

Also I don't know if you will get a github.release object any time the gh action is run

Reply:
https://github.com/geobretagne/mviewer/runs/1357200081?check_suite_focus=true it seems your right. Maybe splitting into 2 files I'll look at it.